Makefile-test.mk

```diff
@@ -55,7 +55,17 @@ endif
 
 # Folder where the e2e tests are located.
 E2E_TARGET ?= ./test/e2e/...
-E2E_KIND_VERSION ?= kindest/node:v1.32.0
+E2E_K8S_VERSION ?= 1.32
+ifeq ($(E2E_K8S_VERSION),1.30)
+	E2E_K8S_VERSION_PATCH ?= 10
+else ifeq ($(E2E_K8S_VERSION),1.31)
+	E2E_K8S_VERSION_PATCH ?= 6
+else ifeq ($(E2E_K8S_VERSION),1.32)
+	E2E_K8S_VERSION_PATCH ?= 3
+else
+	E2E_K8S_VERSION_PATCH ?= 0
+endif
+E2E_KIND_VERSION ?= kindest/node:v$(E2E_K8S_VERSION).$(E2E_K8S_VERSION_PATCH)
 # E2E_K8S_VERSIONS sets the list of k8s versions included in test-e2e-all
 E2E_K8S_VERSIONS ?= 1.30.9 1.31.5 1.32.1
```

Comment:
We're using it in the test-e2e-all target, but I don’t think we actually use the `test-e2e-all` target anywhere.

Reply:
If we don't use it I would just drop it.